docs/getting_started/installation.rst

```diff
@@ -4,217 +4,145 @@
 
-- `pcraster <https://pcraster.geo.uu.nl>`_ This package is used for the reading and writing of pcr maps, which is the file format used by the old Wflow software (written in Python). It is therefore required when you want to convert Wflow models from the Python version to the Julia version. Note that the pcraster package is only available on conda-forge which works if you install hydromt-wflow using Pixi or Conda. If you install hydromt-wflow through pypi you will not be able to access this functionality.
-- `gwwapi <https://github.com/global-water-watch/gww-api>`_ This package is used for providing more resources about reservoirs monitored by the Global Water Watch project. This package is only available through pypi at the moment.
-- `hydroengine <https://github.com/openearth/hydro-engine>`_ Similar to ``gwwapi`` the ``hydroengine`` package gives access to more data sources, and is currently only available through pypi.
-- `wradlib <https://github.com/wradlib/wradlib>`_ Provides downloading and processing functionalities of radar weather data and interpolation functions. Available in both conda-forge and pypi.
-- `pyet <https://github.com/pyet-org/pyet>`_ Provides processing functionalities of several methods to calculate evapotranspriation. Available in both conda-forge and pypi.
-- `gdal <https://gdal.org/en/stable/>`_ Provides many drivers and GIS transformations.  Only available through conda-forge.
+- `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_
+- `Miniforge (Mambaforge) <https://conda-forge.org/docs/>`_
+- `uv <https://docs.astral.sh/uv/>`_
 
-Since some dependencies are only available through conda-forge and some only through pypi, you will need a packange manager that can handle both.
+Both conda variants come preconfigured with the **conda-forge** channel, which provides free and open packages used by HydroMT.
 
-Installation
-============
+Installing HydroMT-Wflow
+========================
 
-There are various ways in which one can install hydrom_wflow in a python environment, the main being:
+HydroMT-Wflow is available from both **PyPI** and **conda-forge**.
+The simplest and most flexible approach is to install it using **pip** inside a new environment.
 
-- :ref:`Pixi <user_install_pixi>`
-- :ref:`Miniforge/conda <user_install_conda>`
+Installation using uv / pip
+----------------------------
 
-All of these options will be discussed below but we recommend using pixi, as it is the fastest and most
-robust (and reproducible) option.
-
-.. _user_install_pixi:
-
-Installation using Pixi
------------------------
-
-.. Tip::
-
-    This is our recommended way of installing HydroMT-Wflow!
-
-
-If you do not have a ``pyproject.toml`` yet you can make one by executing the command:
+We recommend creating a clean environment to avoid dependency conflicts. For example:
 
 .. code-block:: console
 
-    $ pixi init --format pyproject myproject
+    $ conda create -n hydromt-wflow uv python=3.11
```

Comment:
Much nicer and easier I hope! Maybe now python 3.14 is here (and install does not yet work) but this means we will probably not test for 3.11 for long. Shall we recommend 3.13?